.github/workflows/ci.yml

```diff
@@ -349,6 +349,15 @@ jobs:
           sudo BDIR=$BDIR rm -rf ${BDIR}
           test $TEST_RC -eq 0 || /bin/false
 
+      # Log the status of this VM to investigate issues like
+      # https://github.com/containerd/containerd/issues/4969
+      - name: Host Status
+        if: always()
```

Comment:
How do you fill about `if: failure()`, so we execute this step only if something went wrong?

Reply:
I think this should remain `always()`. Useful for comparing failing VMs from passing VMs.